module uim.ai.exceptions;

public {
    import uim.ai.exceptions.exception;
}